### DISCUSSION TITLE
to improve

### DISCUSSION THREAD

**Discussion by mohanad-tammam** [woocommerce/woocommerce#48575] [Checkout Flow]
You must open the modification on the payment gateway so that we can set the data priorities as we want and how can you set the payment via Apple Pay alone without putting it with the payment options, which makes the customer not pay attention to the shipping details and the country to which it is sent. You must make more modifications and improvements so that we can continue our work smoothly.

**Comment by opr** [comment]
Hi, thanks for joining the discussion about the Cart and Checkout blocks! Could you expand on some of your points a little more?

> You must open the modification on the payment gateway so that we can set the data priorities as we want

What do you mean by "set the data priorities as we want"?

> how can you set the payment via Apple Pay alone without putting it with the payment options, which makes the customer not pay attention to the shipping details and the country to which it is sent

Can you explain this in some more detail? Are you talking about the express payment option? Usually this would use the saved address in the user's browser and they wouldn't need to interact with the form any further. Do you mean something else?

> You must make more modifications and improvements so that we can continue our work smoothly.

If you provide clear descriptions of the problems you're trying to solve we can do our best to help you find ways of doing that. It's really important that you give us a detailed description of exactly what it is you're trying to do.

**Comment by mohanad-tammam** [comment]
Dear Thomas
Have a nice day
First of all I would like to apologize for the delay in replying to you
As shown in pictures 1 & 2, I cannot move the quick payment option and make
it with other options such as paying by credit card, and this distracts
customers in purchasing because not
everyone knows that they can store shipping information in Apple Pay
settings as an example, and I will show you another picture of what I want
the payment gateway to look like.
As shown in the other image that was taken from one of the sites and it
appears here that the payment methods are organized under each other, which
does not distract customers.
Thank you, dear Thomas, for your interest in developing the platform
further.
[image: WhatsApp Image 2024-06-29 at 02.54.47_3882e1cf.jpg]
[image: Screenshot 2024-06-29 023037.png][image: Screenshot 2024-06-29
023618.png]

‫في الثلاثاء، 25 يونيو 2024 في 11:27 م تمت كتابة ما يلي بواسطة ‪Thomas
Roberts‬‏ ***@***.***‬‏>:‬

**Reply by opr** [reply]
Hi @mohanad-tammam thanks for your reply - unfortunately the images you linked did not show up, I think it's because you replied via email? Instead maybe you could come directly to GitHub via this link: https://github.com/woocommerce/woocommerce/discussions/48575 to reply?

> As shown in pictures 1 & 2, I cannot move the quick payment option and make
it with other options such as paying by credit card, and this distracts
customers in purchasing because not
everyone knows that they can store shipping information in Apple Pay

Are you saying that you want to disable the express payment options, or move other non-express payment methods _into_ the express payment methods area?

**Reply by mohanad-tammam** [reply]
I will attach the pictures to you to make my point clearer.
![Screenshot 2024-06-29 023037](https://github.com/woocommerce/woocommerce/assets/173177870/13f90b5d-1861-4569-a972-a8115542cee5)
![Screenshot 2024-06-29 023618](https://github.com/woocommerce/woocommerce/assets/173177870/f8362913-9b95-4a46-a571-ef04ff081567)
![WhatsApp Image 2024-06-29 at 02 54 47_3882e1cf](https://github.com/woocommerce/woocommerce/assets/173177870/4aa6c89a-352d-44a2-ac7c-95622dc55eb2)
